charts/tekton-pipelines/templates/buildpacks/tasks/_buildpack.yaml

```diff
@@ -99,6 +99,11 @@ spec:
       type: string
       description: environment name of the app being built, i.e. dev/staging/prod
 
+    - name: sourcemaps_dir
```

Comment:
why did you add it here? I don't see it's used in this task?

Reply:
This is used in `build-static` steps (which are defined from `values.yaml`), so there at the end of the `build-static` step we store static contents in `sourcemaps` dir to be able to use it later at `sentry-release` task (after build process)

Reply:
Pls add that as inline comment or make multiline description (even better) with that last comment.

Reply:
done